resources/checklist-quality.md

```diff
@@ -0,0 +1,45 @@
+This issue is created to collect comments about the rolling release of "______"
+
+This metric can be found here: https://github.com/chaoss/xxxxxxx/xxxxxxx/xxxxxxx.md
+
+See all release candidates: https://chaoss.community/metrics/
+
+## CHAOSS Metric Quality Checklist
+This checklist is used for new and updated metrics to ensure we follow CHAOSS quality standards and processes. Below checklist items don’t have to be completed all at once: create the metric release candidate issue first and then start working on the checklist.
+
+### Process
+
+- [ ] Create “review issue” in the authoring WG’s repo for comments during review period and paste this template in
+- [ ] Create pull request to edit or add metric to WG’s repo (after checking Content Quality and Technical Requirements below)
+- [ ] Add the new metric or metric edit to release notes issue in working group repo
+- [ ] Update spreadsheet of metrics 
+- [ ] Create issue in CHAOSS/Translation repository  to kick-off translation to other languages (please use the the translation issue template)
+- [ ] "Metric Candidate Release" label added to the metric release candidate issue.
+- [ ] Metric was added to website
+- [ ] Announce new/updated metric on mailing list, newsletter, community Zoom call, and Twitter. This can be coordinated with the community manager.
```

Comment:
```suggestion

**When above steps are completed:**

- [ ] Announce new/updated metric on mailing list, newsletter, community Zoom call, and Twitter. This can be coordinated with the community manager.
```